src/test/java/uk/gov/hmcts/reform/cwrdapi/controllers/StaffRefDataControllerTest.java

```diff
@@ -60,17 +70,50 @@ void setUp() {
         MockitoAnnotations.openMocks(this);
     }
 
+    @Test
+    void should_return_service_skills_with_status_code_200() {
+        StaffWorkerSkillResponse staffWorkerSkillResponse =
```

Comment:
can add couple of skills to staffWorkerSkillResponse and then assert the size and content

Reply:
done

Reply:
@shivajihmcts i dont see new changes, did you push your changes ?